cloudmarker/clouds/gcpcloud.py

```diff
@@ -93,10 +100,13 @@ def _get_firewalls(self):
             # Prepare the request to get the list of all firewall rules for a
             # project and execute that request.
             firewall_rules_object = firewall_resource.list(
-                project=self._project_name,
+                project=projectId,
                 pageToken=next_page_token).execute()
 
-            firewall_rules.extend(firewall_rules_object['items'])
+            # firewall_rules_object contains firewall rules for the project
+            # if it has the 'items' key.
+            if 'items' in firewall_rules_object.keys():
+                firewall_rules.extend(firewall_rules_object['items'])
```

Comment:
The double-lookup of `'items'` in `firewall_rules_object` can be avoided with something like this:

```python
firewall_rules.extend(firewall_rules_object.get('items', []))
```